contents/handbook/company/docs.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Documentation
+sidebar: Handbook
+showTitle: true
+---
+
+## Responsibilities
+
+- **Product teams** are responsible for ensuring their docs are up-to-date. This means:
+  - Documenting new features when they're launched.
+  - Correcting mistakes reported by users.
+  - Clarifying documentation based on user feedback and support tickets.
```

Comment:
```suggestion
  - Clarifying documentation based on user feedback and support tickets.
  - Ensuring public betas have documentation which is linked to from feature preview menu
```

Reply:
This is often missed and will avoid people making excuses about 'Oh I didn't document it because it was in beta' or anything